googlecdn.js

```diff
@@ -31,6 +31,26 @@ function isFunction(fn) {
   return typeof(fn) === 'function';
 }
 
+function getUrl(item, type, version) {
```

Comment:
I find this function a bit confusing. The name doesn't really describe what it does and it's only used in one place. Any other way to do this?